app/controllers/concerns/response.rb

```diff
@@ -0,0 +1,5 @@
+module Response
```

Comment:
I'm not sure about this name... what about `ResponseManager`? @GuilleLeopold thoughts?

Reply:
Mmm, I think I prefer something like `act_as_json_request`or `responsable` as it's the most common way to call a concern

Reply:
What should I do then ? 🤔

Reply:
@GuilleLeopold is right, the common way would be `responsable`, @matiasmansilla1989 ?

Reply:
@matiasmansilla1989 ... cri cri

Reply:
We decide to move this logic to the `application_controller`